src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarMain.tsx

```diff
@@ -15,9 +15,8 @@
  */
 
 import React, { useState } from 'react';
-import { connect } from 'react-redux';
+import { useSelector } from 'react-redux';
 
-import { RootState } from 'src/store';
 import { getBreakpointWidth } from 'src/global/globalSelectors';
 import { BreakpointWidth } from 'src/global/globalConfig';
```

Comment:
Not part of this PR but can we separate this import as well?

Reply:
That's done